From b7d107a0156f3c7e07524175d9d27e2d868b58ff Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Thu, 16 Jun 2011 12:33:01 -0400
Subject: [PATCH] dconf: make show-actions lock down optional

The key is no longer necessary in recent versions of
gnome-power-manager.
---
 data/make-dconf-override-db.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/make-dconf-override-db.sh b/data/make-dconf-override-db.sh
index ca04852..252f6ef 100755
--- a/data/make-dconf-override-db.sh
+++ b/data/make-dconf-override-db.sh
@@ -11,7 +11,7 @@ eval `dbus-launch --sh-syntax`
 
 trap 'rm -rf $TMPDIR; kill $DBUS_SESSION_BUS_PID' ERR
 
-gsettings set org.gnome.power-manager show-actions false
+gsettings set org.gnome.power-manager show-actions false || :
 
 gsettings set org.gnome.desktop.a11y.keyboard enable true
 gsettings set org.gnome.desktop.background show-desktop-icons false
-- 
1.7.5.1

